articles/azure-vmware/create-placement-policy.md

```diff
@@ -213,6 +210,9 @@ Azure VMware Solution provides a VMware private cloud in Azure. In this managed
 
 Further, the lack of the desired granularity in the vSphere privileges presents some challenges when managing the placement of the workloads on the private cloud. For example, vSphere DRS rules commonly used on-premises to define affinity and anti-affinity rules can't be used as-is in a VMware Cloud environment, as some of those rules can block day-to-day operation the private cloud. Placement Policies provides a way to define those rules using the Azure VMware Solution portal, thereby circumventing the need to use DRS rules. Coupled with a simplified experience, they also ensure that the rules don't impact the day-to-day infrastructure maintenance and operation activities. 
 
+###  What is the difference between the VM-Host affinity policy and Restrict VM Movement?
+
+A VM-Host affinity policy is used to restrict the movement of VMs to a group of hosts included in the VM-Host affinity policy. Hence, a VM can be vMotioned within the set of hosts selected in the VM-Host affinity policy. On the other hand, Restrict VM movement ensures that the selected VM stays on the host that it currently resides on.
```

Comment:
Change the second and third sentences in line 215 as follows:

Thus, a VM can be vMotioned within the set of hosts selected in the VM-Host affinity policy. Alternatively, **Restrict VM movement** ensures that the selected VM remains on the host on which it currently resides.